ui/src/app/app.module.ts

```diff
@@ -12,7 +12,6 @@ import {CohortBuilderComponent} from 'app/views/cohort-builder/component';
 import {LoginComponent} from 'app/views/login/component';
 import {RepositoryService} from 'app/services/repository.service';
 import {SelectRepositoryComponent} from 'app/views/select-repository/component';
-import {SignInComponent} from 'app/views/sign-in/component'
```

Comment:
These weren't checked in, and weren't needed, so I removed them to make the code compile.